app/content/react/library-reference/widgets.md

```diff
@@ -224,13 +284,12 @@ You can control the legend options through the following properties that must be
 
   const { myLayer } = useSelector((state) => state.carto.layers);
   const source = useSelector((state) => selectSourceById(state, myLayer?.source));
-  const cartoLayerProps = useCartoLayerProps({ source });
+  const cartoLayerProps = useCartoLayerProps({ source, layerConfig });
```

Comment:
`layerConfig` shouldn't use constant layerConfig because it doesn't change!!

```suggestion
  const cartoLayerProps = useCartoLayerProps({ source, layerConfig: myLayer });
```

Reply:
Updated to use `myLayer`.